common/Cargo.toml

```diff
@@ -24,6 +24,7 @@ static_assertions = "1.1"
 strum = { version = "0.24", features = ["derive"] }
 thiserror = "1.0"
 hex = "0.4"
+rand = "0.8.5"
```

Comment:
Please remove rand, and use crypto::random instead. We unified the source of randomness to simplify reviews.

Reply:
The mempool no longer uses rand, but other some parts of `common` still use it..

Reply:
Your PR seems to be moving this dependency from dev-dependencies to dependencies. Can you please fix that?

Reply:
Thanks. Done.